### DISCUSSION TITLE
Docker compose: Unsupported config option for home-assistant: 'network_mode'

### DISCUSSION THREAD

**Discussion by marcoferroha** [indomus/forum#1985] [HUB: Home Assistant]
Buongiorno a tutti,
se uso docker run funziona tutto perfettamente, viceversa se uso il compose ricevo l'errore:

Unsupported config option for home-assistant: 'network_mode'

qualche dritta?

**Comment by snakuzzo** [comment]
Ti ho già risposto sul blog.
Probabilmente perché stai usando il plugin **compose** con il comando `docker compose` e non l'utility `docker-compose`.

Guarda la sintassi sulla [doc di docker](https://docs.docker.com/compose/compose-file/compose-file-v3/).

Probabile tu debba usare network: al posto di network_mode: (dipende anche dalla [file version](https://docs.docker.com/compose/compose-file/compose-file-v3/#compose-and-docker-compatibility-matrix))
